src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/EntityPublished/changeRequest.html.twig

```diff
@@ -32,7 +32,17 @@
                     {% for index, pathUpdate in changeRequest.pathUpdates %}
                     <tr>
                             <td>{{ index }}</td>
-                            <td>{{ pathUpdate }}</td>
+                            <td>
+                            {% if pathUpdate is iterable %}
+                                <ul>
+                                    {%  for name, value in pathUpdate %}
+                                        <li>{{ name }}: {{ value }}</li>
```

Comment:
Name would usually be a numeric index. And that makes little or no sense to the end user I suppose. I think we can get away with just echoing the value here.

Reply:
Indeed meaningless index